pkg/pdutil/pd.go

```diff
@@ -242,32 +251,81 @@ func (p *PdController) getRegionCountWith(
 	return 0, err
 }
 
-// RemoveScheduler remove pd scheduler.
-func (p *PdController) RemoveScheduler(ctx context.Context, scheduler string) error {
-	return p.removeSchedulerWith(ctx, scheduler, pdRequest)
+// PauseScheduler remove pd scheduler temporarily.
+func (p *PdController) PauseScheduler(ctx context.Context, scheduler string) error {
```

Comment:
i find it strange that we create 6 timers and 6 channels for each scheduler, instead of just 1 timer to control pause schedulers at each tick. can we change this to do "batch pause" like

```suggestion
func (p *PdController) PauseSchedulers(ctx context.Context, scheduler []string) error {
```

Reply:
yes, I change it to "batch pause", PTAL again.